static/src/javascripts-legacy/projects/commercial/modules/dfp/empty-advert.js

```diff
@@ -1,13 +1,12 @@
 define([
-    'bonzo',
     'fastdom'
-], function (bonzo, fastdom) {
+], function (fastdom) {
     return emptyAdvert;
 
     function emptyAdvert(advert) {
         fastdom.write(function () {
             window.googletag.destroySlots([advert.slot]);
-            bonzo(advert.node).remove();
+            advert.node.parentNode.removeChild(advert.node);
```

Comment:
I think you could use `advert.node.remove()` in this case, if we add `Element.prototype.remove` to polyfill.io.